core/trino-spi/src/main/java/io/trino/spi/connector/RowChangeParadigm.java

```diff
@@ -34,4 +34,12 @@ public enum RowChangeParadigm
      * corresponding merge processor class is {@code DeleteAndInsertMergeProcessor}.
      */
     DELETE_ROW_AND_INSERT_ROW,
+
+    /**
+     * A paradigm that translates a changed row into a delete by rowId, and an insert of a
```

Comment:
Seems it the wrong description - i.e. this paradigm doesn't delete&insert changes, but updates them in place.